tooling/nargo_cli/tests/comptime_correctness.rs

```diff
@@ -0,0 +1,139 @@
+mod common;
+
+use std::sync::LazyLock;
+use std::{cell::RefCell, collections::BTreeMap};
+
+use acvm::{FieldElement, acir::native_types::WitnessStack};
+use nargo::{foreign_calls::DefaultForeignCallBuilder, ops::execute_program};
+use noirc_abi::input_parser::InputValue;
+use proptest::prelude::*;
+
+static NUM_CASES: LazyLock<u32> = LazyLock::new(|| {
+    std::env::var("COMPTIME_CORRECTNESS_TEST_NUM_CASES").unwrap_or("1".into()).parse().unwrap_or(1)
```

Comment:
```suggestion
    std::env::var("NOIR_COMPTIME_CORRECTNESS_TEST_NUM_CASES").unwrap_or("1".into()).parse().unwrap_or(1)
```

Just so every env var is easy to recognise where it belongs to, or list all NOIR_ env vars.